python/ray/rllib/agents/agent.py

```diff
@@ -14,6 +14,10 @@
 from ray.rllib.utils import FilterManager, deep_update, merge_dicts
 from ray.tune.registry import ENV_CREATOR, _global_registry
 from ray.tune.trainable import Trainable
+from ray.tune.logger import UnifiedLogger
+from ray.tune.result import DEFAULT_RESULTS_DIR
+from datetime import datetime
```

Comment:
Small style comments: `from datetime import datetime` and `import tempfile` should go before Ray imports on line 9.